### PR TITLE
[freerdp] Fix x64-osx triplet

### PR DESCRIPTION
**Pull request by AlexAltea**
`freerdp:x64-osx` fails due to issues in the MacFreeRDP build script, not our portfile.

The rest of FreeRDP builds successfully in macOS, including the client libraries. Since the client **executable** (not the libraries!) is of little use in the vcpkg environment, I think it's convenient to disable its generation (if `VCPKG_TARGET_IS_OSX`) until the problems are fixed on their side.

### CODE REVIEW

**Review by JackBoosY — CHANGES_REQUESTED** [review]
Please update the version info. See [documentation](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md#update-the-version-field-in-the-control-file-of-any-modified-ports).

**Comment by AlexAltea** [comment]
Fixed.

File: ports/freerdp/CONTROL [hunk 1]
```diff
@@ -1,5 +1,5 @@
 Source: freerdp
-Version: 2.0.0-1
+Version: 2.0.0-2
 Homepage: https://github.com/FreeRDP/FreeRDP
 Description: A free implementation of the Remote Desktop Protocol (RDP)
 Build-Depends: openssl, glib (!windows)
```
Comment (JackBoosY) [inline ports/freerdp/CONTROL:5]:
```suggestion
Build-Depends: openssl, glib (!windows)
Supports: !(arm&uwp)
```

File: scripts/ci.baseline.txt [hunk 2]
```diff
@@ -476,7 +476,6 @@ freeglut:x64-uwp=fail
 freeglut:x64-osx=fail
 freerdp:arm64-windows=fail
 freerdp:arm-uwp=fail
```
Comment (JackBoosY) [inline scripts/ci.baseline.txt:478]:
```suggestion
```

File: scripts/ci.baseline.txt [hunk 3]
```diff
@@ -476,7 +476,6 @@ freeglut:x64-uwp=fail
 freeglut:x64-osx=fail
 freerdp:arm64-windows=fail
```
Comment (JackBoosY) [inline scripts/ci.baseline.txt:477]:
```suggestion
```

File: scripts/ci.baseline.txt [hunk 4]
```diff
@@ -476,7 +476,6 @@ freeglut:x64-uwp=fail
 freeglut:x64-osx=fail
 freerdp:arm64-windows=fail
 freerdp:arm-uwp=fail
-freerdp:x64-osx=fail
 freerdp:x64-uwp=fail
```
Comment (JackBoosY) [inline scripts/ci.baseline.txt:479]:
```suggestion
```

**Review by JackBoosY — CHANGES_REQUESTED** [review]

**Review by JackBoosY — APPROVED** [review]

File: ports/freerdp/CONTROL [hunk 5]
```diff
@@ -1,8 +1,9 @@
...
+Supports: !(arm&uwp)
```
Comment (JackBoosY) [inline ports/freerdp/CONTROL:6]:
```suggestion
Supports: !(arm|uwp)
```

**Review by JackBoosY — CHANGES_REQUESTED** [review]

**Review by JackBoosY — APPROVED** [review]

**Comment by ras0219-msft** [comment]
LGTM once the baseline has been sorted

**Review by ras0219-msft — APPROVED** [review]

**Comment by AlexAltea** [comment]
I tried to fix the `ci.baseline.txt` conflict from GitHub editor but it seems the line endings got messed up.
Let me fix that locally in few hours.

**Comment by AlexAltea** [comment]
@ras0219-msft Fixed!

**Comment by JackBoosY** [comment]
Let me test the feature first.

**Comment by JackBoosY** [comment]
`freerdp:x64-osx:    cascade: 8688591b0b3050249fefeab7e0349c7733733870`

Since the freerdp dependency glib does not yet support osx, freerpd is not currently tested in osx.

**Comment by ras0219-msft** [comment]
Thanks for the PR!

**Pull request merged**